packages/components/src/components/@deprecated/input/controller.ts

```diff
@@ -46,7 +47,8 @@ export class InputController extends ControlledInputController implements Watche
 	}
 
 	public validateHideLabel(value?: boolean): void {
-		watchBoolean(this.component, '_hideLabel', value);
+		a11yHint('Property hide-label for inputs: Only use for exceptions like search inputs that are clearly identifiable by their context.');
```

Comment:
Der Hint sollte nur bei `true` angezeigt werden. Dazu einfach den Hints in einen `after`-Hook packen und prüfen, ob der `component.state._hideLabel === true` ist.